modules/gitops-namespace-scoped-instance.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * multitenancy/multitenancy-support-in-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-namespace-scoped-instance_{context}"]
+= Namespace-scoped instance (Application delivery instance)
+
+After you create an Argo CD custom resource (CR) in one of your namespaces, the {gitops-shortname} Operator brings up an Argo CD in this namespace which you can use for application delivery. In its initial state, this instance has permission to deploy resources only in the same namespace it is installed to. However, there are times when organizations have specific requirements to satisfy that require additional tuning.
+
+With the {gitops-shortname} Operator, you can extend the permissions of the Argo CD instance to enable the Argo CD Application Controller to deploy resources into other namespaces apart from where it is installed. 
+
+[NOTE]
+====
+The roles that the {gitops-shortname} Operator in the namespace creates are namespace-scoped and only have access to namespace resources. It cannot perform any actions outside of the namespaces it manages.
+====
+
+.How does this method work?
+
+The {gitops-shortname} Operator allows OpenShift users to instantiate Argo CD instances in their namespaces, as long as they have permission to create Argo CD resources in the `argoproj.io/v1alpha1` API in their namespace. Currently, a namespace-scoped instance has full administrative permissions for one or many namespaces it manages, similar to allowing all verbs for all resources within that namespace.
```

Comment:
or `argoproj.io/v1beta1` . We support both the versions with `v1beta` being the default.

Reply:
Thanks for the feedback. I have addressed it in the documentation.